src/interactions/scrollInteraction.ts

```diff
@@ -0,0 +1,48 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Interaction {
+  export class Scroll extends Interaction.AbstractInteraction {
+    private _mouseDispatcher: Dispatcher.Mouse;
+    private _scrollCallback: (p: Point, deltaAmount: number) => any;
+
+    public _anchor(component: Component.AbstractComponent, hitBox: D3.Selection) {
+      super._anchor(component, hitBox);
+      this._mouseDispatcher = Dispatcher.Mouse.getDispatcher(<SVGElement> this._componentToListenTo.content().node());
+      this._mouseDispatcher.onWheel("Interaction.Scroll" + this.getID(), (p: Point, e: WheelEvent) => this._handleScrollEvent(p, e));
+    }
+
+    private _handleScrollEvent(p: Point, e: WheelEvent) {
+      var translatedP = this._translateToComponentSpace(p);
+      if (this._isInsideComponent(translatedP)) {
+        if (this._scrollCallback) {
+          var deltaPixelAmount = e.deltaY * (e.deltaMode ? 120 : 1);
```

Comment:
Is there testing for this functionality?

Also it'd be good if we could find a reference for this 120...